src/components/App.tsx

```diff
@@ -3185,14 +3185,57 @@ class App extends React.Component<ExcalidrawProps, AppState> {
       file?.type === "application/excalidrawlib.json" ||
       file?.name.endsWith(".excalidrawlib")
     ) {
+      // compares two library items and returns true if they are
+      // the same otherwise returns false.
+      const itemsEqual = (
+        item1: NonDeleted<ExcalidrawElement>[],
+        item2: NonDeleted<ExcalidrawElement>[],
+      ) => {
+        if (item1?.length !== item2?.length) {
+          return false;
+        }
+
+        for (const [index, el] of item1.entries()) {
+          if (!item1[index] || !item2[index]) {
+            return false;
+          }
+
+          if (
+            el.versionNonce !== item2[index].versionNonce &&
+            el.id !== item2[index].id
```

Comment:
this should use `||` instead of `&&`

Reply:
I changed it. but what is the purpose of `versionNone`?

Reply:
To check whether the same element (`id`) isn't different (newer).